tests/Adapters/DatabasePendingApprovalRetrieverTest.php

```diff
@@ -119,4 +131,26 @@ public function testExcludesPageWithoutTheCategoriesOfTheApprover(): void {
 		);
 	}
 
+	public function testReturnsNoPendingApprovalsAfterApprovingAnUnapprovedPage(): void {
+		$approverId = 1;
+		$this->approverRepository->setApproverCategories( $approverId, [ 'Foo' ] );
+
+		$page = $this->createPageWithCategories( [ 'Foo' ] );
+
+		$this->insertApprovalLogEntry( $page->getId(), true, '20240705000000' );
+
+		$pendingApprovals = $this->retriever->getPendingApprovalsForApprover( $approverId );
+		$this->assertCount( 0, $pendingApprovals );
+
+		$this->insertApprovalLogEntry( $page->getId(), false, '20240706000001' );
+
+		$pendingApprovals = $this->retriever->getPendingApprovalsForApprover( $approverId );
+		$this->assertCount( 1, $pendingApprovals );
+
+		$this->insertApprovalLogEntry( $page->getId(), true, '20240706000002' );
+
+		$pendingApprovals = $this->retriever->getPendingApprovalsForApprover( $approverId );
+		$this->assertCount( 0, $pendingApprovals );
+	}
+
```

Comment:
I needed to hardcode the timestamps here. `insertApprovalLogEntry()` calls `$this->db->timestamp()` but the test is too fast and they all return the same timestamp.

Debugging the timestamps in the test:
![Screenshot_20240705_144632](https://github.com/ProfessionalWiki/PageApprovals/assets/1428594/e7138789-187c-42a2-a1f8-7b3be2b4fb92)